e2e-spa/package.json

```diff
@@ -4,33 +4,36 @@
     "private": true,
     "scripts": {
         "dev": "vite --port 4444 --clearScreen false",
-        "compile-definition": "esno scripts/compile-definition",
-        "clean": "rimraf ./src/namespace/index.ts",
+        "gulp-ts": "gulp --require esbuild-register -f gulpfile.ts",
+        "compile-definition": "pnpm gulp-ts -- compileDefinition",
+        "clean": "pnpm gulp-ts -- clean",
+        "test": "pnpm gulp-ts -- test",
```

Comment:
This particular command is not intended to be executed from CI. It is an internal `e2e-spa` command that is a part of global monorepo's `pnpm check-code-integrity` command. It is not called `test:all` because it includes linting & type-checking.